src/rust/bitbox02/src/lib.rs

```diff
@@ -121,14 +123,17 @@ pub fn workflow_confirm(
         }
     };
 
+    let mut params: confirm_params_t = confirm_params_t {
+        title: title_cstr.as_ptr() as *const _,
+        body: body_cstr.as_ptr() as *const _,
+        font: font,
+        scrollable: false,
+        longtouch: longtouch,
+        accept_only: accept_only,
+    };
+
     unsafe {
-        bitbox02_sys::workflow_confirm(
-            title_cstr.as_ptr() as *const _,
-            body_cstr.as_ptr() as *const _,
-            font,
-            longtouch,
-            accept_only,
-        )
+        bitbox02_sys::workflow_confirm(&mut params)
```

Comment:
Doesn't it work to have a non-mutable reference to params?

```
bitbox02_sys::workflow_confirm(&params)
```

Reply:
Totally does, thanks!

Force pushed.